pom.xml

```diff
@@ -154,7 +154,6 @@
                       <detectOfflineLinks>false</detectOfflineLinks>
                       <!--<skip>${skipJavadoc}</skip>-->
                       <use>false</use>
-                      <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
it's required if the javadoc isn't perfect, and the javadoc is never perfect because writing perfect javadoc is horribly annoying.